tools/docker/Dockerfile

```diff
@@ -16,10 +16,9 @@ RUN pip install ansible && \
     ansible-galaxy collection install benthomasson.eda
 COPY . $WORKDIR
 COPY --from=ui-builder /app/ui/dist/ ./ui/dist
-RUN mkdir sqlite && chown -R $USER_ID ./
```

Comment:
please, maintain the `chown -R $USER_ID ./ ` only the `mkdir` should be removed.